File: Iniciante/URI 1008.cs
using System; 

class URI {

    public static void Main(string[] args) { 
        
        int NUMBER, TIME;
        double SALARY, AUX;
        
        NUMBER = Int32.Parse(Console.ReadLine());
        TIME = Int32.Parse(Console.ReadLine());
        AUX = Double.Parse(Console.ReadLine());
        
        SALARY= (AUX * TIME);
        
        Console.WriteLine("NUMBER = {0}", NUMBER);
        Console.WriteLine("SALARY = U$ {0:F2}", SALARY);

    }
}

File: Iniciante/URI 1008.c
#include <stdio.h>

int main()
{

  int NUMBER, TIME;
  double SALARY, AUX;

  scanf("%d", &NUMBER);
  scanf("%d", &TIME);
  scanf("%lf", &AUX);

  SALARY = (AUX * TIME);

  printf("NUMBER = %d\n", NUMBER);
  printf("SALARY = U$ %.2lf", SALARY);
  printf("\n");

  return 0;
}